tests/integration_tests/run_no_reprocessing/test.py

```diff
@@ -46,7 +46,7 @@ def test_integration_2step_saltproc_no_reproc_heavy(setup):
     test_fuel_mdens = test_result['MAT_fuel_MDENS'][:, -1]
 
     test_mdens_error = np.array(ref_fuel_mdens - test_fuel_mdens)
-    np.testing.assert_array_equal(test_mdens_error, ref_mdens_error)
+    np.testing.assert_array_almost_equal(test_mdens_error, ref_mdens_error)
```

Comment:
Why are you saying `almost equal` here? Is there some rounding error between values of the array here?

Reply:
I use this because monte-carlo codes are stochastic and can have slightly different results each time you run them, which would cause a `assert_equal` statement to fail.

Reply:
So this test is based on some result out of Serpent or OpenMC then? If that's the case, can you specify a random seed? Or does doing that still have some small variation in the result?

Reply:
The default sensitivity of `assert_almost_equal` is on the order of 10^-6, which I think is sufficiently small since we are comparing errors, not base values.

Reply:
That makes sense. And this test passes?

Reply:
Yes